catalog-info.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-idea-indexing-metrics-collector
+  description: Buildkite Pipeline for idea-indexing-metrics-collector
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/idea-indexing-metrics-collector
+
+spec:
+  type: buildkite-pipeline
+  system: buildkite
```

Comment:
Hi @breskeby 👋🏼,
When checking the Backstage logs, I identified  that the `spec.owner` property is missing, and therefore, this file does not contain a valid definition.
Could you please add it? it should be something like `owner: group:cloud-team`  if the team owner of this repo is the **cloud-team**. Otherwise, please replace it with the proper value.
On the other hand, we are working to [fix this bug here](https://github.com/elastic/catalog-info/pull/168), so after this PR gets merged, future **catalog-info.yaml** file definition will not contain this bug.
Thank you very much 😊

Reply:
For further reference, here you go a better example of the [owner declaration](https://github.com/elastic/catalog-info/blob/5cebb19014df7b18d1dd0fb7fdbf24a43e6a5f54/catalog-info.yaml#L45).